ledger-api/grpc-definitions/com/daml/ledger/api/v1/ledger_offset.proto

```diff
@@ -11,11 +11,25 @@ option java_package = "com.daml.ledger.api.v1";
 option csharp_namespace = "Com.Daml.Ledger.Api.V1";
 
 // Describes a specific point on the ledger.
+//
+// The Ledger API endpoints that take offsets allow to specify portions
+// of the ledger that are relevant for the client to read.
+//
+// Offsets returned by the Ledger API can be used as-is (e.g.
+// to keep track of processed transactions and provide a restart
+// point to use in case of need).
+//
+// The format of absolute offsets is opaque to the client: no
+// client-side transformation of an offset is guaranteed
+// to return a meaningful offset.
+//
+// The server implementation ensures internally that offsets
+// are lexicographically comparable.
 message LedgerOffset {
 
   oneof value {
-    // Absolute values are acquired by reading the transactions in the stream.
-    // An offset is an opaque string that can be compared according to lexicographical ordering.
+    // Absolute values are acquired by reading from the TransactionService.
```

Comment:
And the ACS and `GetLedgerEnd`?

Reply:
Indeed.

But as this shows, listing the endpoints where this is returned is only prone to becoming stale (and does not really provide the user with meaningful information, as the endpoints are most likely the starting point from where the documentation for offsets are reached, not the other way around).

I addressed that by simply removing this line in https://github.com/digital-asset/daml/pull/6465/commits/cc8e123d84b0eb54df026b044cee10c7d690ddfd.

Reply:
Thanks! Removing makes sense :+1: